src/swaps/boltz.rs

```diff
@@ -589,10 +589,12 @@ pub struct MrhResponse {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct Webhook {
+pub struct SubmarineWebhook {
```

Comment:
Instead of having three types for webhook which only differ in `pub status: Option<Vec<SubSwapStates>>`, couldn't we just use a generic for the possible states?

Reply:
Good point, updated to use a generic type for the Webhook status list